src/transactions/OfferExchange.cpp

```diff
@@ -710,14 +710,26 @@ applyPriceErrorThresholds(Price price, int64_t wheatReceive, int64_t sheepSend,
             }
         }
     }
-    else
+    else if (round != RoundingType::PATH_PAYMENT_STRICT_SEND)
```

Comment:
maybe move to a case/switch here for clarity (one has to read the comment inside the "else" case to understand what's going on)

Reply:
Done.